File: reni-backend/src/main/java/pe/gob/mimp/reni/dto/request/MaestraCargaMasivaRequest.java
package pe.gob.mimp.reni.dto.request;

import java.io.Serializable;
import java.util.List;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
public class MaestraCargaMasivaRequest implements Serializable {
	private static final long serialVersionUID = 1L;

	private List<MaestraCargaMasivaDetalleRequest> listaMaestra;
	private List<String> listaMaestraIds;
	private Long idUsuarioCrea;
}

File: reni-frontend/src/main/reni-frontend/src/app/modules/intranet/dto/request/maestra-carga-masiva.request.ts
import { MaestraCargaMasivaDetalleRequest } from "./maestra-carga-masiva-detalle.request";

export class MaestraCargaMasivaRequest {
    listaMaestra: MaestraCargaMasivaDetalleRequest[];
    listaMaestraIds: string[];
    idUsuarioCrea: number;
}